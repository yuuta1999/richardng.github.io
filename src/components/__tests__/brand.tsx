import React from 'react'
import { render, waitForElement, cleanup } from '@testing-library/react'
import 'jest-styled-components'

import Brand from '../brand'

afterEach(cleanup)

test('it works', async () => {

  const { getByText, container } = render(<Brand>Richardng.com</Brand>)


  expect(container.firstChild).toMatchSnapshot()

  await waitForElement(() => getByText(/Richardng.com/i))
})
