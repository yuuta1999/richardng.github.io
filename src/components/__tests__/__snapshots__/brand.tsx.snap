// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it works 1`] = `
.c0 {
  display: inline-block;
  padding-top: 0.3125rem;
  padding-bottom: 0.3125rem;
  margin-right: 0.5rem;
  line-height: inherit;
  white-space: nowrap;
  color: var(--Theme-NavLink--Color);
  font-size: 1.25rem;
  font-weight: 700;
}

.c0:hover,
.c0:focus {
  color: var(--Theme-NavLink--Color__onHover);
  -webkit-text-decoration: none;
  text-decoration: none;
}

<a
  class="c0"
>
  Richardng.com
</a>
`;
